bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java

```diff
@@ -283,7 +283,9 @@ void parseYamlFile(String fileName, String fileBody) {
       this.cloudScope = fileBody.contains(CLOUD_AUTH_SCOPE);
 
       // API Serivce config has Locations API.
-      this.containsLocations = fileBody.contains(LOCATIONS_MIXIN);
+      if (fileBody.contains(LOCATIONS_MIXIN)) {
+        this.containsLocations = true;
```

Comment:
I'm assuming the default value is `false` :) (from what I remember about Java, this assumption is correct)

Reply:
Yep, it is :)